modules/metering-reports.adoc

```diff
@@ -262,6 +262,31 @@ spec:
   reportingEnd: "2019-07-31T00:00:00Z"
 ----
 
+[id="metering-expiration_{context}"]
+== expiration
+
+Add the `expiration` field to set a retention period on a scheduled metering Report. You can avoid manually removing the Report by setting the `expiration` duration value. The retention period is equal to the Report `creationDate` plus the `expiration` duration. The Report is removed from the cluster at the end of the retention period if no other Reports or ReportQueries depend on the expiring Report.
```

Comment:
@lbarbeevargas @bentito should we mention the valid units for the `expiration`?  For example, is `seconds` a valid unit?

Reply:
Go duration units, https://golang.org/src/time/format.go?s=40605:40651#L1364, so pasting from there:
```
Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". 
```

Reply:
Good question! I can put the valid values in a callout under the yaml example. @bentito, I forget the valid values you said during our meeting about the feature.

Reply:
@lbarbeevargas @bentito & @timflannagan1  pointed out during today's scrum that the expiration time might not be exact depending on how the api objects are cached (they can elaborate on that).  Realistically though, it probably won't make sense to have the `expiration` period < 1m

Reply:
I think it could use a warning that this is not precise timing, meant to work on the order of several minutes, not nanoseconds.